sdk/messaging/eventgrid/azsystemevents/ci.yml

```diff
@@ -0,0 +1,36 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/messaging/eventgrid/azsystemevents
+      - eng/
```

Comment:
Was this intentional?

Reply:
The `eng/`? It was in my original - is it not supposed to be there?

Reply:
It depends on if you want this module to be included in CI when changes to `/eng` are made. We have a handful (`azcore`, `internal`, and `aztemplate` to name a few). I'd opt to remove it, unless there's something interesting/unique here (e.g. some dependency on tooling under `/eng`) that would cause us to want to validate it whenever `/eng` is updated.

Reply:
Gone!